qiskit_bot/api.py

```diff
@@ -149,7 +149,16 @@ def on_pull_event(data):
     if data['action'] == 'opened':
         repo_name = data['repository']['full_name']
         pr_number = data['pull_request']['number']
+        pr = REPOS[repo_name].gh_repo.get_pull(pr_number)
         if repo_name in REPOS:
+            if data['pull_request']['author_association'] != 'MEMBER':
+                # tag PR with 'community PR' label & notify reviewers
+                labels = pr.get_labels()
+                label_names = [label.name for label in labels]
+                if "Community PR" not in label_names:
```

Comment:
Does this label exist on every repo that qiskit-bot is running on? Do you think there is value in making this a per repo configuration so that the label use can be customized.

Reply:
Personally I would prefer to keep this hardcoded as I would like to enforce consistent naming across the different Qiskit repos for community focused things. So if a contributor wants to get started contributing to any of the qiskit repos they can expect GFI, HW and CPR labels to mean the same thing whichever repo they end up in. Also makes our lives easier as reviewers when we start taking on more community reviewer tasks for other repos